{"uuid": "9a96c2f2-01ef-4aad-9205-1e1edee3e874", "children": ["dd455b8a-8ecb-45c9-96fa-36a30f0fee14"], "befores": [{"name": "__init__", "status": "broken", "statusDetails": {"message": "FileNotFoundError: 配置文件D:\\售后自动化\\UIPOMTest\\project\\TEST\\env\\test\\config.ini不存在！\n", "trace": "  File \"D:\\app\\Python10\\lib\\site-packages\\pluggy\\_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n  File \"D:\\app\\Python10\\lib\\site-packages\\_pytest\\fixtures.py\", line 1126, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"D:\\app\\Python10\\lib\\site-packages\\_pytest\\fixtures.py\", line 932, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"D:\\售后自动化\\UIPOMTest\\project\\TEST\\test_case\\conftest.py\", line 15, in __init__\n    ini = ReadConfig(pro_name, pro_env)\n  File \"D:\\售后自动化\\UIPOMTest\\libs\\common\\read_config.py\", line 13, in __init__\n    raise FileNotFoundError(\"配置文件%s不存在！\" % INI_PATH)\n"}, "start": 1658421703131, "stop": 1658421703302}], "start": 1658421703131, "stop": 1658421705786}